modules/security-context-constraints-about.adoc

```diff
@@ -132,6 +152,16 @@ The `restricted` SCC:
 The restricted SCC is the most restrictive of the SCCs that ship by default with the system. However, you can create a custom SCC that is even more restrictive. For example, you can create an SCC that restricts `readOnlyRootFS` to `true` and `allowPrivilegeEscalation` to `false`.
 ====
 
+|`restricted-v2`
+| Like the `restricted` SCC, but with the following differences:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
+
+By default, this SCC replaces the `restricted` SCC for new clusters.
+
```

Comment:
@stlaz Would be be very helpful if we could add an WARNING/NOTE here clarifying that solutions which are consider restricted for Kubernetes might not be qualified to restricted-v2 with an example:

> Differences between OCP and K8S constraints
Being admitted to the restricted-v2 SCC is sufficient to qualify for restricted PSA admission.  The inverse is not true.  A pod might qualify for PSA-restricted, but not for the restricted-v2 SCC.  The most obvious example of this is a pod that specifies runAsUser=1001.  Since the pod specifies a non-zero runAsUser, it qualifies as psa-restricted.  However the restricted-v2 SCC requires that pods RunAsRange meaning the uid must be within the range of allowed UIDs for the namespace, and 1001 will not be within that range.

Reply:
I think any mention of the relationship between SCCs and PSa should appear in the PSa section, not here.

Reply:
Can be done as a follow-up